ro_help/hub/forms.py

```diff
@@ -75,8 +75,12 @@ class Meta:
         ]
         widgets = {
             "email": EmailInput(),
-            "city": forms.TextInput(
-                attrs={"class": "cityAutoComplete", "data-url": reverse_lazy("city-autocomplete"),}
+            "city": forms.Select(
+                attrs={
+                    "class": "cityAutoComplete", 
+                    "disabled": "true", 
+                    "data-url": reverse_lazy("city-autocomplete")
+                }
```

Comment:
This could be problematic as it will load all 13K cities in the page select. It will make loading the registration page slower and the Code4ro partners who provided this data would not be be happy about leaving it out in the open to be scraped. (my understanding was that they are very protective of it)